.github/workflows/release.yaml

```diff
@@ -1,7 +1,12 @@
 name: Release
 on:
   push:
-    tags: [ 'v*.*.*' ]
+    # Only release on supported semantic version tagging e.g. v0.0.1-rc.0
+    tags:
+      - 'v[0-9]+.[0-9]+.[0-9]+'
```

Comment:
Should the . be escaped?

Reply:
Glob patterns aren't regex. They don't get escaped: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
TIL